docs/selectors.md

```diff
@@ -84,9 +84,45 @@ Id engines are selecting based on the corresponding atrribute value. For example
 
 ### zs
 
-ZSelector is an experimental engine that tries to make selectors survive future refactorings. Example: `zs=div ~ "Login"`.
+Z-selector is an experimental engine to define selectors that can **survive future refactorings**. Changes in the UI markup can often cause existing UI selectors to break. While `data-*` attributes can be useful to establish contracts between app and test code, they can be an additional maintenance burden. It can also be difficult to introduce new attributes if you are testing third-party UI.
 
-TODO: write more.
+Z-selector aims to search elements with precision and resiliency against layout changes. It does not depend on strict nesting, and can localize element search through accessible selectors, like text content and type of element. It can then combine that with precise last-mile lookup.
+
+Z-selector selector uses the `~` combinator 
```

Comment:
Should we consider using a character that isn’t already a CSS combinator?

Reply:
I hope nobody uses it. In CSS definition ~ claims they are siblings, in zs case ~ claims they have common parent. I would not overinvest into it, we'll be investing into zs when we look at selector generation more.

Reply:
Is it common parents or common ancestors? The reason I ask is that if we think of zs as a superset of CSS, then it seems simpler to not overload an existing CSS concept.